test/data/html/node.mustache

```diff
@@ -275,18 +275,20 @@ function run_html_tests(test_obj, Urlencoded, js_beautify, html_beautify, css_be
             '  body {background-color:lightgrey}\n' +
             '  h1   {color:blue}\n' +
             '</style>');
-        opts.unformatted = unformatted;
+        opts.content_unformatted = content_unformatted;
 
-        unformatted = opts.unformatted;
-        opts.unformatted = ['custom-element'];
+        inline_tags = opts.inline_tags;
+        opts.inline_tags = ['custom-element'];
         test_fragment('<div>should <custom-element>not</custom-element>' +
                       ' insert newlines</div>',
                       '<div>should <custom-element>not</custom-element>' +
                       ' insert newlines</div>');
-        opts.unformatted = unformatted;
+        opts.inline_tags = inline_tags;
 
-        // Tests that don't pass, but probably should.
-        // bth('<div><span>content</span></div>');
+        bth('<div><span>content</span></div>');
+
+        test_fragment('<p>Should remove <q><span \n\nclass="some-class">attribute</span></q> newlines</p>',
```

Comment:
Instead of  just this one test, add a whole new section showing the accurate behavior of inlines.  
You also need to show them behaving correctly in line-wrap situations.